Repo.rb

```diff
@@ -217,21 +214,25 @@ def num_independent_pending_commits
 end
 
 class RepoInfo
-  attr_reader :path_to, :subrepos, :is_head, :bindings
+  # path_to: String = Absolute Path to .repo file
+  # subrepos: [RepoInfo] = Array of subrepos
+  # is_head: Bool = Whether or not this is a head repo (no parents)
+  attr_reader :path_to, :subrepos, :is_head
 
-  def initialize(path_to: , subrepos: [], is_head: true, bindings: {})
+  def initialize(path_to: , subrepos: [], is_head: true)
     @path_to = path_to
     @subrepos = subrepos
     @is_head = is_head
-    @bindings = bindings
   end
 
+  # Convert to hash, encoding subrepos as path to their repofile
+  # Used for serialization to disk - so that a .repo doesn't
+  # hold information about its repo's grandchildren
   def to_hash
-    subrepos_hash = @subrepos.map(&:to_hash)
-    {"path_to" => @path_to,
-     "subrepos" => subrepos_hash,
-     "isHead" => @isHead,
-     "bindings" => @bindings}
+    { "path_to" => @path_to,
+      "subrepos" => @subrepos.each{ |sub| sub.path_to },
```

Comment:
Space missing to the left of {.<br>Pass `&:path_to` as an argument to `each` instead of a block.